src/proxy-ui-api/src/main/resources/openapi-definition.yaml

```diff
@@ -3235,23 +3243,18 @@ paths:
           description: an existing item already exists
         '500':
           description: internal server error
-  /timestamping-services/{url}:
     delete: # ok
       tags:
         - security server
       summary: delete timestamping service
       operationId: deleteTimestampingService
       description: Administrator removes timestamping service.
-      parameters:
-        - in: path
-          name: url
-          description: url of the timestamping service
-          required: true
-          schema:
-            type: string
-            format: text
-            minLength: 1
-            maxLength: 255
+      requestBody:
+        description: Timestamping service to delete
+        content:
+          application/json:
+            schema:
+              $ref: '#/components/schemas/TimestampingService'
```

Comment:
Removed `updated_at`, `message` and `status` from `TimestampingService`, and created a new `TimestampingServiceDiagnostics` objects with the attributes since they're used in the diagnostics only.